README.md

```diff
@@ -139,6 +146,13 @@ const MyPage = observer(() => {
     props => new MyPageViewModel(props, userStore), // Factory-function will be called only once on first render
     { userId } // These `props` are reactive. It will be passed and updated in view-model every time it changes without creating new instance of view-model 
   );
+  
+  //** or
+  // const viewModel = useViewModel(
+  //    MyPageViewModel, 
+  //   { userId },
+  //   [userStore] // Extra dependencies for the view-model constructor
```

Comment:
Maybe I don't know some details, but why would this extra functionality be useful?) Why can't you just pass it on as part of `props`:

```tsx
const viewModel = useViewModel(
  MyPageViewModel, 
  { userId, userStore },
...
```

Reply:
The thing is that `props` is reactive data that can be updated on the next render. But dependencies like `userStore` or, for example, `authService` are not data. They're not supposed to be a part of observable `props` and their responsibility is different. So, we separate it explicitly in view-model constructor params, and in `useViewModel` function signature as well.